.github/workflows/java-ci.yml

```diff
@@ -223,3 +209,7 @@ jobs:
           elif [ -f "gradlew" ]; then
             ./gradlew dockerPushNative --no-daemon
           fi
+
+      - name: Cleanup images
+        if: ${{ always() }}
+        run: docker image prune -f
```

Comment:
For my own benefit: why is this needed?

Reply:
A few days ago we had `no space left on device` issues on self hosted github runner because we had large number of created images, so it I thought it would be good to delete those images each time we create it and publish to the docker registry. The same is already done here: https://github.com/oracle-quickstart/oci-micronaut/blob/43b87a6021dd876d609f0e3ead6df294e028cddc/.github/workflows/java-ci.yml#L144